support-figma/extended-layout-plugin/src/localization-module.ts

```diff
@@ -394,6 +398,21 @@ function normalizeString(characters: string): string {
       // Preserver tabs at the beginning and the end
       // Use '\t' for precise translations
       .replace(/\t/g, "\\t")
+      // Replace the following dash characters with unicode for precise translations
+      // - hyphen-minus
+      .replace(/\u002d/g, "\\u002d")
+      // - hyphen
+      .replace(/\u2010/g, "\\u2010")
+      // - non-breaking hyphen
+      .replace(/\u2011/g, "\\u2011")
+      // - figure dash
+      .replace(/\u2012/g, "\\u2012")
+      // - n dash
+      .replace(/\u2013/g, "\\u2013")
+      // - m dash
+      .replace(/\u2014/g, "\\u2014")
+      // - minus sign
+      .replace(/\u2212/g, "\\u2212")
```

Comment:
Where did you get this list of characters to replace?

Reply:
I found it on wiki. I used one of the dash occasionally in the styled text run test and Android Studio suggests to use unicode for those dashes when I checked the gen_strings.xml.